docs/learn/learn-simple-payouts.md

```diff
@@ -34,16 +37,22 @@ which takes 28 eras. If a validator were to immediately chill and start unbondin
 calculated, and nobody issued a payout for that era from that validator in the next 28 eras, the
 reward would no longer be claimable.
 
-:::info In order to be absolutely sure that staking rewards can be claimed, users should trigger a payout before 28 eras have passed.
+:::info In order to be absolutely sure that staking rewards can be claimed, users should trigger a
+payout before 28 eras have passed.
 
 :::
 
 Anyone can trigger a payout for any validator, as long as they are willing to pay the transaction
 fee. Someone must submit a transaction with a validator ID and an era index. Polkadot will
-automatically calculate that validator's reward, find the top {{ polkadot_max_nominators }}
+automatically calculate that validator's reward, find the top
+{{ polkadot: <RPC network="polkadot" path="query.staking.maxNominatorsCount" defaultValue={50000}/> :polkadot }}
```

Comment:
I think the Wiki presented this information in an incorrect way earlier. When someone submits a transaction with a validator ID and an era index and that particular validator is oversubscribed, only the top 256 nominators get rewarded pro-rata

<img width="1430" alt="image" src="https://user-images.githubusercontent.com/86818441/188880861-cd95e23c-5266-44ec-b134-d6d160984c18.png">

Reply:
Good catch!  I swapped these few instances out for the appropriate endpoint.  Should be good to go now.